File: EASY/ONBOARDING/c#/onboarding.cs
using System;

class Player
{
    static void Main(string[] args)
    {
        while (true)
        {
            string enemy1 = Console.ReadLine(); // name of enemy 1
            int dist1 = int.Parse(Console.ReadLine()); // distance to enemy 1
            string enemy2 = Console.ReadLine(); // name of enemy 2
            int dist2 = int.Parse(Console.ReadLine()); // distance to enemy 2

            if (dist1 < dist2)
                Console.WriteLine(enemy1);
            else
                Console.WriteLine(enemy2);
        }
    }
}

File: EASY/ONBOARDING/js/onboarding.js
while (true)
{
    var enemy1 = readline(); // name of enemy 1
    var dist1 = parseInt(readline()); // distance to enemy 1
    var enemy2 = readline(); // name of enemy 2
    var dist2 = parseInt(readline()); // distance to enemy 2

    if (dist1 < dist2)
        print(enemy1);
    else
        print(enemy2);
}